api/environments/dynamodb/dynamodb_wrapper.py

```diff
@@ -27,22 +27,25 @@
 logger = logging.getLogger()
 
 
-class DynamoWrapper:
+class BaseDynamoWrapper:
     table_name: str = None
 
     def __init__(self):
         self._table = None
-        if self.table_name:
+        if table_name := self.get_table_name():
             self._table = boto3.resource(
                 "dynamodb", config=Config(tcp_keepalive=True)
-            ).Table(self.table_name)
+            ).Table(table_name)
 
     @property
     def is_enabled(self) -> bool:
         return self._table is not None
 
+    def get_table_name(self) -> str:
+        return self.table_name
```

Comment:
I added this as it simplifies some of the mocking in tests that have built on top of this in my branch to add the overrides endpoint.

Reply:
sigh

Reply:
Well, I was finding that the class was being loaded before the settings (using SettingsWrapper fixture) were applied. I guess I could use patch.object to set the table name directly, but I don't love using patch.object where it can be avoided.

Reply:
Like I said this morning though, I want to refactor my tests so I can rip this out for now if you'd rather?

Reply:
> I can rip this out for now if you'd rather?

I've done this. Will re-add in my branch if I need it.